pkg/controller/installation/products/launcher/reconciler.go

```diff
@@ -0,0 +1,405 @@
+package launcher
+
+import (
+	"context"
+	"github.com/RHsyseng/operator-utils/pkg/olm"
+	launcherv1alpha2 "github.com/fabric8-launcher/launcher-operator/pkg/apis/launcher/v1alpha2"
+	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/marketplace"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/config"
+	"github.com/integr8ly/integreatly-operator/pkg/resources"
+	appsv1 "github.com/openshift/api/apps/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	appsv1Client "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
+	pkgerr "github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	defaultInstallationNamespace        = "launcher"
+	defaultSubscriptionName             = "integreatly-launcher"
+	defaultLauncherName                 = "launcher"
+	defaultLauncherConfigMapName        = "launcher"
+	launcherRouteName                   = "launcher"
+	clientId                            = "launcher"
+	githubOauthAppCredentialsSecretName = "launcher-oauth-github"
```

Comment:
@matskiv We've created this secret in the Launcher namespace as the console would break if this was not provided. This may have changed in the new release, double checking that now. :+1:

Reply:
@matskiv This has been fixed with the new release, can now delete the github oauth secret as its no longer required by the console :smiley:

Reply:
Thanks @matskiv I will test and see.

Reply:
Thanks @matskiv the latest release of the launcher-operator doesn't need this.